src/panels/Menu/TopMenuBar/Menus/FileMenu.tsx

```diff
@@ -0,0 +1,48 @@
+import { Kbd, Menu, Text } from '@mantine/core';
+import { modals } from '@mantine/modals';
+
+import { useGetBoolPropertyValue, useOpenSpaceApi } from '@/api/hooks';
+import { AddFileIcon, ConsoleIcon, ExitAppIcon } from '@/icons/icons';
+
+import { TopBarMenuWrapper } from '../TopBarMenuWrapper';
+
+export function FileMenu() {
+  const luaApi = useOpenSpaceApi();
+
+  const [isConsoleVisible, setIsConsoleVisible] =
+    useGetBoolPropertyValue('LuaConsole.IsVisible');
```

Comment:
Do we want to use a key here instead?

Reply:
I think not. There are multiple places where we are still using just the URI for the properties (e.g. in helpmenuHelpMenu, WwtProvider, SceneGraphNodeMetaInfo), so I think we can just leave it here for now

It's a future discussion to have, though. I personally find it easier to identify which property is begin  accessed when the URI is used rather than a key name